src/hooks/usePrevious.ts

```diff
@@ -0,0 +1,9 @@
+import { useEffect, useRef } from "react";
+
+export function usePrevious(value: any) {
```

Comment:
Synes denne er kanskje litt for generisk, kunne det vært vært mer beskrivende med `usePreviousElement` eller noe sånn? Jeg måtte gå gjennom hvor det ble brukt for å skjønne konteksten.

Reply:
Renamer til usePreviousValue